app/views/devise/registrations/start.html.slim

```diff
@@ -1,11 +1,12 @@
 - title t('titles.registrations.start')
 
-h1.h3.my0 = t('headings.create_account')
+h1.h3.my0 = t("headings.create_account_#{service_provider}", sp: @sp_name)
```

Comment:
+1 to moving the conditional to a helper.

The SP-vs-not-SP is a very high level template logic decision. If we ever get to the point where we are maintaining lots of templates for specific SPs, then I think we should revisit the interpolation idea, but based on my understanding of the product future, that's unlikely.